packages/global/package.json

```diff
@@ -35,6 +35,7 @@
     ]
   },
   "dependencies": {
+    "@blocksuite/icons": "=2.1.0-alpha.7",
```

Comment:
Where is the upstream repository? I am concerned if we hold private dependencies.

Reply:
https://github.com/toeverything/icons